CMakeLists.txt

```
@@ -207,27 +207,25 @@ execute_process(
   OUTPUT_VARIABLE PG_VERSION_STRING
   OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-if (NOT ${PG_VERSION_STRING} MATCHES "^PostgreSQL[ ]+([0-9]+)\\.([0-9]+)(\\.([0-9]+))*")
+if (NOT ${PG_VERSION_STRING} MATCHES "^PostgreSQL[ ]+([0-9]+)(\\.|rc|devel)([0-9]+)?")
```

I think you might want something like this:
```suggestion
if (NOT ${PG_VERSION_STRING} MATCHES "^PostgreSQL[ ]+([0-9]+)(rc|devel|\\.([0-9]+))")
```

---

that wouldnt work for RC versions but i've adjusted the regex slightly